rxredux/build.gradle.kts

```diff
@@ -77,7 +77,7 @@ val javadocJar by tasks.creating(Jar::class) {
     dependsOn(doc)
     from(doc)
 
-    classifier = "javadoc"
```

Comment:
`classifier` is deprecated and recommended to use `archiveClassifier.set` instead.